### ISSUE TITLE
Send to self doesn't show receiver

### ISSUE THREAD

**Issue by dexX7** [OmniLayer/omniexplorer#152]
In the following transaction no explicit referenceaddress is specified for a simple send, which results in a valid send to self. However, OE doesn't show it as recipient:

https://www.omniexplorer.info/tx/bfd1d982c137beb23035838505505265e836d12ae423943c5fc07f566a94f14a

On a protocol level there is indeed no referenceaddress, so I'd lean not to change the result on the RPC level.

**Comment by dexX7** [comment]
The shown balance on the address overview is not correct, because these 991.90 USDT where subtracted, but not added back to the sender internally, as it seems:

https://www.omniexplorer.info/address/1BUJGjEqoCK46yk5uuw1T35jAzSFbpGr4F

**Comment by achamely** [comment]
@dexX7 Yes I can see. The issue is that all the data used by OE is fed to the database from the output of Omnicore's rpc. 
If omnicore didn't give a 'destination' we had no way to interpret it. 

With regards to the rpc, I disagree. At a protocol level a tx can be interpreted/represented/encoded (class a, b, c) in different ways however the rpc output is the same. A simple send should present the information to the rpc interface in the same way.  Either this is a valid simple send which should populate the reference address or it is a invalid send.  Integrators look for consistency when it comes to rpc outputs.
